### PR TITLE
Delete service message

### PR DESCRIPTION
**Pull request by pataruco**
### Name of the story

solves #431 

### Acceptance Criteria

When a "delete file" message is published to service bus delete queue, the Delete Service is watching so that it picks up the message.

This code that enable reading is [here](https://github.com/redbadger/AzureSDKForRust/tree/service-bus-read-event)

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: medicines/doc-index-updater/src/service_bus_client/mod.rs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    let policy_name = std::env::var("CREATE_QUEUE_POLICY_NAME")
+        .expect("Set env variable CREATE_QUEUE_POLICY_NAME first!");
+
+    let policy_key = std::env::var("CREATE_QUEUE_POLICY_KEY")
+        .expect("Set env variable CREATE_QUEUE_POLICY_KEY first!");
+
+    Client::new(service_bus_namespace, queue_name, policy_name, policy_key)
+}
+
+pub async fn delete_service_worker() -> Result<String, AzureError> {
```
Comment (TimboTambo) [inline medicines/doc-index-updater/src/service_bus_client/mod.rs]: Is the intention to move this worker to the delete service module?

File: medicines/doc-index-updater/Cargo.toml [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 version = "0.1.0"
-authors = ["Stuart Harris <stuart.harris@red-badger.com>", "Robin James Kerrison <robinjames.kerrison@red-badger.com>", "Matt Doughty <matt.doughty@red-badger.com>"]
+authors = ["Stuart Harris <stuart.harris@red-badger.com>", "Robin James Kerrison <robinjames.kerrison@red-badger.com>", "Matt Doughty <matt.doughty@red-badger.com>", "Pedro Martin <pedro.martin@red-badger.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+azure_sdk_service_bus = { git = "https://github.com/redbadger/AzureSDKForRust", branch = "service-bus-read-event" }
+azure_sdk_core = { git = "https://github.com/redbadger/AzureSDKForRust", branch = "service-bus-read-event" }
```
Comment (TimboTambo) [inline medicines/doc-index-updater/Cargo.toml]: We can update these to reference the original repo now that your PR was merged!

Comment (pataruco) [reply medicines/doc-index-updater/Cargo.toml]: On the newest version of [azure_sdk_service_bus](https://crates.io/crates/azure_sdk_service_bus)  and [azure_sdk_core](https://crates.io/crates/azure_sdk_core) are some breakinhg changes.

Let's keep this until we sort it out

File: medicines/doc-index-updater/src/service_bus_client/mod.rs (hunk 1 above)
Comment (pataruco) [reply medicines/doc-index-updater/src/service_bus_client/mod.rs]: Indeed

File: infrastructure/modules/service-bus/main.tf [hunk 3]
```diff
@@ -27,12 +27,27 @@ resource "azurerm_servicebus_queue" "doc_index_updater_service_bus_queue" {
...
+}
+
+
 
 resource "azurerm_redis_cache" "doc_index_updater_redis" {
   name = var.name
 
   capacity            = 0
-  enable_non_ssl_port = true # FIXME: Set to true to enable development / Change this before go to prod
+  enable_non_ssl_port = false # FIXME: Set to true to enable development / Change this before go to prod
```
Comment (craiga) [inline infrastructure/modules/service-bus/main.tf]: Get rid of the `FIXME` comment (I'm assuming this has been fixed).

**Pull request merged**